js/renovation/ui/grids/data_grid_light/widgets/data_row.tsx

```diff
@@ -1,51 +1,91 @@
 import {
-  Component, JSXComponent, ComponentBindings, OneWay,
+  Component, JSXComponent, ComponentBindings, OneWay, Effect, InternalState, Consumer,
 } from '@devextreme-generator/declarations';
+import { combineClasses } from '../../../../utils/combine_classes';
+import { createGetter, PluginsContext, Plugins } from '../../../../utils/plugin/context';
+import { Column, RowData } from '../types';
+
+export type DataRowPropertiesGetterType = (data: RowData) => Record<string, unknown>;
+export const DataRowPropertiesGetter = createGetter<DataRowPropertiesGetterType>(() => ({}));
+export type DataRowClassesGetterType = (data: RowData) => Record<string, boolean>;
+export const DataRowClassesGetter = createGetter<DataRowClassesGetterType>(() => ({}));
 
 export const viewFunction = (viewModel: DataRow): JSX.Element => (
   <tr
-    className="dx-row dx-data-row dx-column-lines"
+    className={viewModel.cssClasses}
     role="row"
     aria-selected="false"
-    // TODO uncomment after https://trello.com/c/kVXfSWI7
-    // aria-rowindex={viewModel.props.rowIndex + 1}
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    {...viewModel.additionalParams}
   >
-    {viewModel.cellTexts.map((text, index) => (
-      <td
-        // eslint-disable-next-line react/no-array-index-key
-        key={index}
-        // TODO uncomment after https://trello.com/c/kVXfSWI7
-        // aria-describedby={`dx-col-${index + 1}`}
-        aria-selected="false"
-        role="gridcell"
-        // TODO uncomment after https://trello.com/c/kVXfSWI7
-        // aria-colindex={index + 1}
-      >
-        {text}
-      </td>
-    ))}
+    {viewModel.props.columns.map((column, index) => {
+      const { cellTemplate: CellTemplate, dataField } = column;
+      return (
+        <td
+          // eslint-disable-next-line react/no-array-index-key
+          key={index}
+          // TODO uncomment after https://trello.com/c/kVXfSWI7
+          // aria-describedby={`dx-col-${index + 1}`}
+          aria-selected="false"
+          role="gridcell"
+        >
+          {
+            CellTemplate
+              ? <CellTemplate data={viewModel.props.data} />
+              : dataField && String(viewModel.props.data[dataField])
```

Comment:
String method inside jsx does not work in angular. So, rewrite it to use getter for example.